### ISSUE TITLE
Deprecation warning about using set-output

### ISSUE THREAD

**Issue by MBilderbeek** [SimenB/github-actions-cpu-cores#25]
I noticed that when using this action, I get a warning about deprecation of set-output:

```
Run SimenB/github-actions-cpu-cores@v1.0.1
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```
Can you fix this, please?

**Comment by SimenB** [comment]
This was actually fixed by #22, but I forgot to build it 😅

**Comment by SimenB** [comment]
https://github.com/SimenB/github-actions-cpu-cores/releases/tag/v1.0.2
